appdatabase/migrations/sql/1652194966_add_notifications_settings_column.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE settings ADD COLUMN notifications_settings BLOB;
```

Comment:
I wonder if we should rather introduce a new table `notification_settings` instead, otherwise doing migrations on this data is going to be hard..

Reply:
Ah, @richard-ramos beat me to it: https://github.com/status-im/status-go/pull/2669#discussion_r869542156

Reply:
@PascalPrecht introducing new table is good way to redo (refactor) settings thing in an appropriate way. I mentioned [here](https://github.com/status-im/status-go/pull/2669#discussion_r869598466) how it looks now.